packages/plugin-conversation/src/interceptor.js

```diff
@@ -187,9 +191,10 @@ export default class ConversationInterceptor extends Interceptor {
    * @returns {Promise<Boolean>}
    */
   shouldEncryptRequest(options) {
-    if (!options.method || options.method.toUpperCase() === `GET`) {
+    if (options.shouldEncryptRequest === false || !options.method || options.method.toUpperCase() === `GET`) {
```

Comment:
it's only there to make a test work, so let's revisit when we fix the horror that is interceptor-modification right now.